design/ciel-tokens-extracted.css

```
@@ -0,0 +1,62 @@
+  --color-ciel-text: #2e2b26; /* ink — NOT #1C1A17 */
+  --color-ciel-accent: #b88a4a; /* warm gold */
+
+  /* Present in the compiled stylesheet but never given a --color-ciel-* name.
+   * Treat as observed usage, not as sanctioned tokens. */
+  --observed-gradient-stop: #e9e1d4;
+  --observed-muted: #a99d8b;
+  --observed-secondary: #756d60;
+
+  /* Typography, verified. */
+  --font-sans:
+    Inter, Satoshi, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont,
```

Stylelint (17.14.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)